tavla/pages/_document.tsx

```diff
@@ -0,0 +1,64 @@
+import Document, {
+    Html,
+    Head,
+    Main,
+    NextScript,
+    DocumentContext,
+    DocumentInitialProps,
+} from 'next/document'
+import { isUnsupportedBrowser } from 'utils/browserDetection'
+
+interface MyDocumentProps extends DocumentInitialProps {
+    needsRefresh: boolean
+}
+
+class MyDocument extends Document {
+    static async getInitialProps(
+        context: DocumentContext,
+    ): Promise<MyDocumentProps> {
+        const initialProps = await Document.getInitialProps(context)
```

Comment:
Fjerne denne?

Reply:
Må visst ha det for å sørge for at all next-funksjonalitet og stiler skal fungere🤷 Docsen til next gir også det i eksempelet